app/javascript/crayons/formElements/Toggles/__stories__/Toggles.stories.jsx

```diff
@@ -0,0 +1,41 @@
+import { h } from 'preact';
+import { Toggle } from '..';
+import ToggleDoc from './Toggles.mdx';
+
+export default {
+  component: Toggle,
+  title: 'Components/Form Elements/Toggles',
+  decorators: [
+    (story) => (
+      <label class="flex gap-2">
+        {story()}
+        Remember: form elements should be wrapped with labels!
```

Comment:
the perfect way to communicate "this component doesn't include a label, but you should" 😄